From 70baf55197c071d70364bcaf40c3a5d45637768b Mon Sep 17 00:00:00 2001
From: Brian Bennett <brian.bennett@joyent.com>
Date: Fri, 12 Apr 2019 22:06:02 -0700
Subject: [PATCH] joyent/triton#252 Typo on README.md where physical install
 indicates COAL download Reviewed by: Trent Mick <trentm@gmail.com> Approved
 by: Trent Mick <trentm@gmail.com>

---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index eaf272b..2afbd11 100644
--- a/README.md
+++ b/README.md
@@ -128,7 +128,7 @@ the [Joyent Manufacturing Database](https://docs.joyent.com/private-cloud/hardwa
 To install Triton, first download the latest release image:
 
 ```bash
-curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/coal-latest.tgz
+curl -C - -O https://us-east.manta.joyent.com/Joyent_Dev/public/SmartDataCenter/usb-latest.tgz
 ```
 
 Once you have downloaded the latest release image, you will need to
-- 
2.21.0

